### PR TITLE
[SHRINKRES-316] Clean up the poms

### PR DESCRIPTION
**Pull request by petrberan**
https://issues.redhat.com/browse/SHRINKRES-316

### CODE REVIEW

File: gradle/impl-gradle-embedded-archive/pom.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
         <dependency>
             <groupId>org.codehaus.plexus</groupId>
             <artifactId>plexus-compiler-javac</artifactId>
-            <version>${version.org.codehaus.plexus.compiler.javac}</version>
```
Comment (xstefank) [inline gradle/impl-gradle-embedded-archive/pom.xml:49]: Wouldn't it be better to keep it version.package.of.dependency?

File: pom.xml [hunk 2]
```diff
@@ -110,43 +121,43 @@
...
                 <artifactId>maven</artifactId>
-                <version>${version.org.apache.maven}</version>
+                <version>${version.maven}</version>
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
             <dependency>
                 <groupId>org.jboss.shrinkwrap</groupId>
                 <artifactId>shrinkwrap-bom</artifactId>
-                <version>${version.org.jboss.shrinkwrap}</version>
```
Comment (xstefank) [inline pom.xml:120]: probably keep like this

**Review by xstefank — APPROVED** [review]
There is an enormous duplication. Maybe in future we could generate these files.

**Comment by petrberan** [comment]
Wrong commit, will resolve tomorrow

**Comment by petrberan** [comment]
Fixed

File: gradle/impl-gradle-embedded-archive/pom.xml [hunk 3]
```diff
@@ -27,7 +27,7 @@
         <dependency>
             <groupId>org.gradle</groupId>
             <artifactId>gradle-tooling-api</artifactId>
-            <version>${version.gradle-tooling-api}</version>
+            <version>${version.org.gradle.gradle-tooling-api}</version>
```
Comment (xstefank) [inline gradle/impl-gradle-embedded-archive/pom.xml:30]: I still think you should go with package, not package.artifact. If we will have some automation in the future we should align this with what you will have in main pom.xml

File: pom.xml [hunk 4]
```diff
@@ -318,6 +331,7 @@
                         <format>html</format>
                         <format>xml</format>
                     </formats>
+                    <check/>
```
Comment (xstefank) [inline pom.xml:334]: what is this?

File: pom.xml [hunk 5]
```diff
@@ -22,34 +22,45 @@
...
+        <version.commons-io.commons-io>2.13.0</version.commons-io.commons-io>
+        <version.eu.maveniverse.maven.mima>2.3.2</version.eu.maveniverse.maven.mima>
+        <version.org.apache.maven.maven>3.9.3</version.org.apache.maven.maven>
+        <version.org.apache.maven.plugin-tools.maven-plugin-annotations>3.5</version.org.apache.maven.plugin-tools.maven-plugin-annotations>
+        <version.org.apache.maven.shared.maven-invoker>3.0.0</version.org.apache.maven.shared.maven-invoker>
+        <version.org.arquillian.spacelift.arquillian-spacelift>1.0.2</version.org.arquillian.spacelift.arquillian-spacelift>
+        <version.org.codehaus.plexus.plexus-classworlds>2.7.0</version.org.codehaus.plexus.plexus-classworlds>
+        <version.org.codehaus.plexus.plexus-compiler-javac>2.7</version.org.codehaus.plexus.plexus-compiler-javac>
+        <version.org.codehaus.plexus.plexus-component-annotations>2.1.0</version.org.codehaus.plexus.plexus-component-annotations>
+        <version.org.codehaus.plexus.plexus-sec-dispatcher>2.0</version.org.codehaus.plexus.plexus-sec-dispatcher>
```
Comment (xstefank) [inline pom.xml:44]: Try to align versions where you can from the same projects.

**Review by xstefank — CHANGES_REQUESTED** [review]
I find it hard to believe you need to have a separate version for every artifact. Align what you can.

File: pom.xml (hunk 4 above)
Comment (petrberan) [reply pom.xml:334]: Check tag is required by the plugin config, in this case it doesn't do anything. This tag makes IDEA not consider it an error anymore

File: pom.xml (hunk 5 above)
Comment (petrberan) [reply pom.xml:44]: The goal of this issue is to clean up the poms, updating versions will be part of the future development

File: gradle/impl-gradle-embedded-archive/pom.xml (hunk 3 above)
Comment (petrberan) [reply gradle/impl-gradle-embedded-archive/pom.xml:30]: Having problem understanding what is the desired outcome for this comment. Can you provide expected version property for - for example - `arquillian-spacelift-api`?

**Review by xstefank — APPROVED** [review]
After @petrberan explained to me that he plans to finish remaining comments in a followup issue. I'm merging this.

**Pull request merged**